### PR TITLE
Include subdomains in allowed audience for credentials

### PR DESCRIPTION
**Pull request by dmitrizagidulin**

### CODE REVIEW

File: test/unit/oidc-manager-test.js [hunk 1]
```diff
@@ -316,4 +317,34 @@ describe('OidcManager', () => {
...
+      expect(oidc.filterAudience(['https://example.com/test', 'https://other.com']))
+        .to.be.true()
+    })
+
+    it('should be true if audience is a subdomain of server uri', () => {
+      expect(oidc.filterAudience('https://alice.example.com/profile#me'))
+        .to.be.true()
+    })
+
+    it('should be false if audience is a different domain than sever', () => {
```
Comment (RubenVerborgh) [inline test/unit/oidc-manager-test.js]: *server

**Comment by RubenVerborgh** [comment]
Looks good to me. The whole `setUserHeader` logic is in node-solid-server, right?

**Comment by dmitrizagidulin** [comment]
> The whole setUserHeader logic is in node-solid-server, right?

Yeah, exactly.

File: test/unit/oidc-manager-test.js (hunk 1 above)
Comment (dmitrizagidulin) [reply test/unit/oidc-manager-test.js]: Thanks! :)

**Pull request merged**